src/lib/gql/GqlHasuraAdmin.ts

```diff
@@ -87,6 +81,26 @@ export class Gql {
     ).circles;
   }
 
+  async getCurrentEpoch(
```

Comment:
Is there something about this that is admin only?

Reply:
yes I should do this

Reply:
does it have to be in both places?